docs/Components-TextField.md

```diff
@@ -0,0 +1,146 @@
+# TextField Component
+
+[Back to all components](./Components.md)
+
+Ignite's `TextField` Component is an enhanced version of the built-in React Native [`TextInput`](https://reactnative.dev/docs/textinput) component. It consists of TextInput and a Text(./Components-Text.md) label. 
+
+With this component you will be able to standardise TextInput component across your app.
+
+```tsx
+import { TextField } from '../components';
+
+const [input, setInput] = useState("")
+<TextField
+  value={input}
+  onChangeText={(value) => setInput(value)}
+  label="Name"
+  placeholder="omg your name"
+  style={$header}
+  inputStyle={$inputStyle}
+  preset="default"
+/>
+
+```
+
+## Props
+
+### `label`
+
+The `label` prop is optional. We encourage you to not use this but rather use the `labelTx` prop instead.
```

Comment:
Is this true? In the case that we don't have a translation available for the user's locale, wouldn't we want the `label` regardless? I'd be curious what other members of the team think about this, I don't have much experience with how we do things.

I'd also add a note that if `label` is missing `labelTx` must be set for any label text to be present.

Reply:
The label prop is optional and tbh I prefer that as I sometimes don't use labels for text field(personal preference) - eg login forms.